doc-content/enterprise-only/admin-and-config/managing-business-central-adding-data-sources-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='managing-business-central-adding-data-sources-proc']
+= Adding a data source to {CENTRAL}
```

Comment:
It looks as if this part repeats itself twice.

Reply:
Corrected.